src/Microsoft.IdentityModel.JsonWebTokens/JwtTokenUtilities.cs

```diff
@@ -429,6 +429,24 @@ private static long ParseTimeValue(JToken jToken, string claimName)
             throw LogHelper.LogExceptionMessage(new FormatException(LogHelper.FormatInvariant(LogMessages.IDX14300, LogHelper.MarkAsNonPII(claimName), jToken.ToString(), LogHelper.MarkAsNonPII(typeof(long)))));
         }
 
+        internal static string SafeLogJwtToken(object obj)
```

Comment:
>object [](http://example.com/codeflow?start=47&length=6)

I thought this was meant for strings only, for any other object type we should be able to use the provided interface

Reply:
@ciaozhang

Reply:
It is the callback func we used in SecurityArtifact struct. The reason we take obj as the type of parameter is LogHelper treat all arguments as object.

Reply:
I see, still regardless of the parameter type this is meant to be consumed by using string version of tokens correct? 

how do you justify the code for non strings that you have here?

Reply:
It will use the default ToString() method return the GetType().ToString()